Gems/Multiplayer/Code/Include/Multiplayer/AutoGen/AutoComponent_Source.jinja

```diff
@@ -722,18 +722,10 @@ void {{ ClassName }}::NotifyChanges{{ AutoComponentMacros.GetNetPropertiesSetNam
     }
 {%         endif %}
 {%     else %}
-
```

Comment:
Oops, this is removing a fix that was added just a few hours ago. We'll want this file to match development (have no changes)
https://github.com/o3de/o3de/commit/1f187c02f4930363973cf4dd9ee85cc48420a545

Reply:
reverted